debugging-with-ssh-access.md

```diff
@@ -0,0 +1,81 @@
+Often the best way to troubleshoot failures in your pipelines is to SSH into a
+job and inspect log files, running processes, and directory paths. Semaphore 2.0
+gives you the option to access all running jobs via SSH, to restart your jobs
+in debug mode, or to start on-demand virtual machines to explore the build
```

Comment:
I would say "the actual build environment".

Reply:
The word `actual` implies that there is an non-actual build environment on Semaphore. Can you give a broader context how the `actual` word makes things clearer?